dir.props

```diff
@@ -69,6 +69,8 @@
     <Platform>AnyCPU</Platform>
     <OutputType>Library</OutputType>
     <RunApiCompat>true</RunApiCompat>
+    <!-- Default to using the open public key -->
+    <AssemblyOriginatorKeyFile>$(ToolsDir)Open.snk</AssemblyOriginatorKeyFile>
```

Comment:
I refactored the keys a little and made those explicitly declare the MSFT key. I also did a full build to ensure nothing under platforms was using the default open key.